server/leaderboard.js

```diff
@@ -27,23 +27,125 @@ var users = require('./users.js');
  *
  * If shrt is true, return leaderboard with max eight entries.
  */
-exports.leaderboard = function(userid, shrt, callback) {
+exports.leaderboard = function(userid, shrt, type, callback) {
     users.getStudentsListSorted(0, function(err, studentlist) {
+
         var rank = 0;
         var last = -1;
         var userind;
 
+        //Sort the list of students depending on the type (defualt sorting is by points)
+        if (type === 'accuracy'){
+            studentlist = studentlist.sort(function(a,b) {
```

Comment:
would be a good idea to use names other than a,b,x, and y

Reply:
I meant to chagne them but I forgot :sweat_smile: